src/fides/api/util/saas_util.py

```diff
@@ -435,6 +435,19 @@ def get_identity(privacy_request: Optional[PrivacyRequest]) -> Optional[str]:
     return identities[0] if identities else None
 
 
+def get_identities(privacy_request: Optional[PrivacyRequest]) -> List[str]:
```

Comment:
GH doesn't let me commented on undiffed lines, but it may be worth removing that raised exception from `get_identity` above (L432-L434) now, since it's not true? i know we're not going to hit it in practice (at least in normal app runtime), but just feels like a vestige that could potentially cause confusion!

Reply:
There's a handful of request overrides that still use the old `get_identity`. I'm leaving that behavior for now to avoid breaking anything else this close to release. Maybe I can address those in a follow up ticket?

Reply:
ah ok understood, all good 👍